### DISCUSSION TITLE
nb更新面板问题

### DISCUSSION THREAD

**Discussion by admimare** [monsterxcn/nonebot-plugin-gspanel#53] [提问]
使用nb面板更新无法更新 后台报错以下内容

[message] 触发 Matcher(type='message', module=nonebot_plugin_gspanel)                   [PyBot][23:15:56.794][INFO] [gspanel]正在查找 UID121114968 的「全部」角色面板..                                     [PyBot][23:15:56.805][INFO] [gspanel.data_source]UID121114968 的角色展柜数据正在刷新！                              [PyBot][23:15:59.308][INFO] [gspanel.data_source]正在为 UID121114968 的 [0]神里绫华/[1]优菈/[2]申鹤/[3]纳西妲/[4]雷 电将军/[5]甘雨/[6]刻晴/[7]夜兰 重新请求伤害计算接口       [PyBot][23:16:02.337][ERROR] [message]Running Matcher(type='message', module=nonebot_plugin_gspanel) failed.        Traceback (most recent call last):                          File "/root/TRSS_Yunzai/Yunzai/plugins/py-plugin/main.py", line 17, in <module>                                       yunzai_nonebot.run()                                    File "/root/TRSS_Yunzai/Yunzai/plugins/py-plugin/yunzai_nonebot/__init__.py", line 31, in run                         nonebot.run(*args, **kwargs)                            File "/root/TRSS_Yunzai/home/.cache/pypoetry/virtualenvs/py-plugin-IA85WoXH-py3.10/lib/python3.10/site-packages/nonebot/__init__.py", line 273, in run                          get_driver().run(*args, **kwargs)                       File "/root/TRSS_Yunzai/Yunzai/plugins/py-plugin/yunzai_nonebot/hijack/driver.py", line 42, in run                    loop.run_until_complete(self.serve())                   File "/usr/lib/python3.10/asyncio/base_events.py", line 636, in run_until_complete                                    self.run_forever()                                      File "/usr/lib/python3.10/asyncio/base_events.py", line 603, in run_forever                                           self._run_once()                                        File "/usr/lib/python3.10/asyncio/base_events.py", line 1909, in _run_once                                            handle._run()                                           File "/usr/lib/python3.10/asyncio/events.py", line 80, in _run                                                        self._context.run(self._callback, *self._args)          File "/root/TRSS_Yunzai/Yunzai/plugins/py-plugin/yunzai_nonebot/hijack/message.py", line 10, in wrapper               return await func(*args, **kwargs)                      File "/root/TRSS_Yunzai/home/.cache/pypoetry/virtualenvs/py-plugin-IA85WoXH-py3.10/lib/python3.10/site-packages/nonebot/message.py", line 141, in _check_matcher                await _run_matcher(Matcher, bot, event, state, stack, dependency_cache)                                         > File "/root/TRSS_Yunzai/home/.cache/pypoetry/virtualenvs/py-plugin-IA85WoXH-py3.10/lib/python3.10/site-packages/nonebot/message.py", line 187, in _run_matcher                  await matcher.run(bot, event, state, stack, dependency_cache)                                                     File "/root/TRSS_Yunzai/home/.cache/pypoetry/virtualenvs/py-plugin-IA85WoXH-py3.10/lib/python3.10/site-packages/nonebot/internal/matcher/matcher.py", line 732, in run          await self.simple_run(bot, event, state, stack, dependency_cache)                                                 File "/root/TRSS_Yunzai/home/.cache/pypoetry/virtualenvs/py-plugin-IA85WoXH-py3.10/lib/python3.10/site-packages/nonebot/internal/matcher/matcher.py", line 707, in simple_run                                                             await handler(                                          File "/root/TRSS_Yunzai/home/.cache/pypoetry/virtualenvs/py-plugin-IA85WoXH-py3.10/lib/python3.10/site-packages/nonebot/dependencies/__init__.py", line 108, in __call__        return await cast(Callable[..., Awaitable[R]], self.call)(**values)                                               File "/root/TRSS_Yunzai/home/.cache/pypoetry/virtualenvs/py-plugin-IA85WoXH-py3.10/lib/python3.10/site-packages/nonebot_plugin_gspanel/__init__.py", line 51, in panel_handle                                                             rt = await getPanel(uid, char)                          File "/root/TRSS_Yunzai/home/.cache/pypoetry/virtualenvs/py-plugin-IA85WoXH-py3.10/lib/python3.10/site-packages/nonebot_plugin_gspanel/data_source.py", line 300, in getPanel                                                             return await template_to_pic(                           File "/root/TRSS_Yunzai/home/.cache/pypoetry/virtualenvs/py-plugin-IA85WoXH-py3.10/lib/python3.10/site-packages/nonebot_plugin_htmlrender/data_source.py", line 231, in template_to_pic                                                   template = template_env.get_template(template_name)     File "/root/TRSS_Yunzai/home/.cache/pypoetry/virtualenvs/py-plugin-IA85WoXH-py3.10/lib/python3.10/site-packages/jinja2/environment.py", line 1010, in get_template              return self._load_template(name, globals)               File "/root/TRSS_Yunzai/home/.cache/pypoetry/virtualenvs/py-plugin-IA85WoXH-py3.10/lib/python3.10/site-packages/jinja2/environment.py", line 969, in _load_template             template = self.loader.load(self, name, self.make_globals(globals))                                               File "/root/TRSS_Yunzai/home/.cache/pypoetry/virtualenvs/py-plugin-IA85WoXH-py3.10/lib/python3.10/site-packages/jinja2/loaders.py", line 126, in load                           source, filename, uptodate = self.get_source(environment, name)                                                   File "/root/TRSS_Yunzai/home/.cache/pypoetry/virtualenvs/py-plugin-IA85WoXH-py3.10/lib/python3.10/site-packages/jinja2/loaders.py", line 218, in get_source                     raise TemplateNotFound(template)

**Comment by admimare** [comment]
trss崽 教程使用了硫酸钡Barite[https://gitee.com/CUZNIL/Yunzai-install](url)的教程

**Reply by admimare** [reply]
py插件的配置
log_level: INFO
command_start:
  - ""
  - "#"
  - /
command_sep:
  - .
nickname:
  - 云崽
  - yunzai
superusers:
  - 2153199921
need_at: no
plugins:
  - nonebot_plugin_ping
  - nonebot_plugin_blacklist
  - nonebot_plugin_memes
  - nonebot-plugin-gspanel
host: 127.0.0.1
port: 43215
independent: false
encoding: utf-8
setup_check: false
gspanel_alias:
  - nb面板
gspanel_scale: 1
resources_mirror: https://enka.network/ui/
resources_dir: /root/TRSS_AllBot/Yunzai/plugins/py-plugin/data
memes_check_resources_on_startup: true
memes_prompt_params_error: true
memes_use_sender_when_no_image: true
求告知该如何解决

**Comment by monsterxcn** [comment]
你这报错是给人看的吗？

**Reply by monsterxcn** [reply]
粗看是 `resources_dir` 填的有问题，路径下的 HTML 渲染模板文件找不到

**Reply by admimare** [reply]
请问该怎么解决呢
报错总共就这么多 我全给复制下来了 可能手机运行的复制下来变了
